pom.xml

```diff
@@ -6,13 +6,14 @@
 
     <groupId>uk.gov.homeoffice.digital.sas</groupId>
     <artifactId>parentpom</artifactId>
-    <version>0.1.0</version>
+    <version>${revision}</version>
     <packaging>pom</packaging>
 
     <properties>
+        <revision>0.1.1</revision>
```

Comment:
Using a custom version property (like `commonVersion`) throws this warning: 
`'version' contains an expression but should be a constant. @ uk.gov.homeoffice.digital.sas:parentpom:${commonVersion}, /<home_folder>/callisto-jparest/pom.xml, line 9, column 14
...
It is highly recommended to fix these problems because they threaten the stability of your build.
For this reason, future Maven versions might no longer support building such malformed projects.
`
Note especially the last statement in the warning message.

The use of revision seems to be the recommended way: https://stackoverflow.com/a/51969067/2219361

Reply:
Fair enough.